build/repo.props

```diff
@@ -16,6 +16,8 @@
     <!-- These properties are use by the automation that updates dependencies.props -->
     <LineupPackageId>Internal.AspNetCore.Universe.Lineup</LineupPackageId>
     <LineupPackageRestoreSource>https://dotnet.myget.org/F/aspnetcore-dev/api/v3/index.json</LineupPackageRestoreSource>
+
+    <GenerateSignRequest>false</GenerateSignRequest>
```

Comment:
What's this for?

Reply:
This is needed with the newest Korebuild (temporarily) @natemcmaster is aware of it.